tools/autoformat.py

```diff
@@ -28,6 +28,9 @@ def _run_cmd(args):
             "--check" if args.check else None,
             "--quiet" if args.quiet else None,
             "--verbose" if args.verbose else None,
+            # default line length should be 80 chars
```

Comment:
Not default, this can't be overridden.

Probably better to say explicitly why it's hardcoded.